<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="NL-DSuresh-W7" name="Default test" tests="1" failures="1" timestamp="25 Oct 2016 13:11:06 GMT" time="1192.177" errors="0">
  <testcase name="getDBData" time="1189.529" classname="DatabasePack.DatabaseConnection">
    <failure type="com.ibm.as400.access.AS400JDBCSQLSyntaxErrorException" message="[SQL0204] ISOR1PF in DURGA type *FILE not found.">
      <![CDATA[com.ibm.as400.access.AS400JDBCSQLSyntaxErrorException: [SQL0204] ISOR1PF in DURGA type *FILE not found.
	at com.ibm.as400.access.JDError.createSQLExceptionSubClass(JDError.java:885)
	at com.ibm.as400.access.JDError.throwSQLException(JDError.java:686)
	at com.ibm.as400.access.AS400JDBCStatement.commonPrepare(AS400JDBCStatement.java:1823)
	at com.ibm.as400.access.AS400JDBCStatement.executeQuery(AS400JDBCStatement.java:2446)
	at DatabasePack.DatabaseConnection.getDBData(DatabaseConnection.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- getDBData -->
</testsuite> <!-- Default test -->
